### ISSUE TITLE
[January] LARMOR/SANS2D/ZOOM/LOQ - sans/scans libraries - harmonize

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#7594]
- Make sure they're using ISISNeutronMuon/instrumentscripts, not a private fork (LARMOR are on a fork?)
- Update to latest master
- Test in conjunction with scientists
- Make sure all instruments are on the same version of the library

The test across multiple instruments has been specifically requested by S.K. (LOQ)

Please extend across to the reflectometers as well (requested by JW)

**Comment by Tom-Willemsen** [comment]
Converted from task to issue as it is non-trivial

**Comment by Tom-Willemsen** [comment]
Having spoken to Max Skoda in a somewhat-related meeting Friday morning, the situation for reflectometers is more complicated and I'm not sure we can fit them in before cycle start. Spun out into separate issue https://github.com/ISISComputingGroup/IBEX/issues/7596

**Comment by Tom-Willemsen** [comment]
PRs:
- https://github.com/ISISNeutronMuon/InstrumentScripts/pull/106
- I have tested these changes on all 4 beamlines, with input from the relevant scientists.

Reviewer please check:
- The 4 SANS instruments (ZOOM, LARMOR, SANS2D, LOQ) are all on the branch listed above
- The branch listed above is up-to-date with master
- There are no local changes present on the instruments

Once the PR above is merged:
- Create a task to checkout `master` rather than this ticket's branch on each instrument, as a task for next shutdown.
